### PR TITLE
add screening protocol

### PR DESCRIPTION
**Pull request by sanuann**

### CODE REVIEW

File: protocol/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          background-image: url(https://raw.githubusercontent.com/ReproNim/demo-protocol/master/VoicePilot/about_the_study.svg);
+          background-position: center;
+          background-size: contain;
+          background-repeat: no-repeat">
+        </div>
+      </section>
+      <br>
+<section>
+<div class="container-fluid">
+  <h2>Welcome to the Demonstration Protocol</h2>
```
Comment (satra) [inline protocol/README.md:15]: This document should be updated to reflect the screening protocol. Also this doc should list pointers to national resources for people who need immediate help.

File: protocol/MentalWellBeingScreening_schema [hunk 2]
```diff
@@ -0,0 +1,83 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc1/contexts/generic",
+        {
+            "rl": "https://raw.githubusercontent.com/ReproNim/reproschema-library/master/activities/"
```
Comment (satra) [inline protocol/MentalWellBeingScreening_schema]: can we start fixing these to checksums

**Comment by sanuann** [comment]
I'll create a new repo to avoid the mess in gh-pages

File: protocol/README.md (hunk 1 above)
Comment (sanuann) [reply protocol/README.md:15]: @danielmlow will be updating it.

**Pull request merged**